src/Symfony/Bundle/TwigBundle/composer.json

```diff
@@ -24,6 +24,7 @@
         "twig/twig": "~1.28|~2.0"
     },
     "require-dev": {
+        "symfony/console": "~3.3",
```

Comment:
well, either FraneworkBundle should require it, or it should not be broken when the component is not installed. Otherwise it is a BC break in FrameworkBundle, and TwigBundle is the wrong place to fix it (all projects depending on FrameworkBundle would have to do the same, which is wrong)